AI/VCAI/FuzzyEngines.cpp

```diff
@@ -378,7 +378,11 @@ float VisitObjEngine::evaluate(Goals::VisitObj & goal)
 		return 0;
 
 	auto obj = ai->myCb->getObj(ObjectInstanceID(goal.objid));
-
+	if(!obj)
```

Comment:
While this solution would fix VisitObj crashes - all crashes appearing here indicate failed logic somewhere else.

Reply:
Yes, but that's what error messages are for. I'd rather not to have crashes in AI even when something fails.